xml/System/String.xml

```diff
@@ -4621,7 +4621,9 @@ In contrast, the use of interpolated strings in the following example produce mu
 
 ## Examples
 
-Numerous examples that call the <xref:System.String.Format%2A> method are interspersed through the [Remarks](#remarks) section of this article. The C# examples are runnable from your browser; just select the **Run** button. 
+Numerous examples that call the <xref:System.String.Format%2A> method are interspersed through the [Remarks](#remarks) section of this article.  
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
not related to this PR but the next paragraph has an issue. Perhaps it should say which are included in a .NET Core...